src/backend/distributed/planner/function_call_delegation.c

```diff
@@ -734,6 +734,7 @@ EnableInForceDelegatedFuncExecution(Const *distArgument, uint32 colocationId)
 						   colocationId));
 	AllowedDistributionColumnValue.distributionColumnValue = copyObject(distArgument);
 	AllowedDistributionColumnValue.colocationId = colocationId;
+	AllowedDistributionColumnValue.executorLevel = ExecutorLevel;
```

Comment:
could we end up re-enterint this logic when calling a pushed down function from a pushed down function?